src/m365/tenant/commands/serviceannouncement/serviceannouncement-healthissue-list.ts

```diff
@@ -0,0 +1,65 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import { ServiceHealthIssue } from '@microsoft/microsoft-graph-types';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  service: string;
```

Comment:
Since `service` is optional, we should denote it as `service?`

Reply:
Yes, this should have a ?